.github/workflows/package.Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM alpine:latest AS builder
```

Comment:
Should we specify `alpine:3.20` to make sure we build on the same version we are currently shipping?

I guess we will always attempt to ship as close to `alpine:latest` for alpine-lima and wsl-distro, so it probably won't matter; idk.

If you do pin the Alpine version, then it should probably be passed in from the workflow and not hardcoded here.

Reply:
I'll consider it in the follow-up (for the actual release flow).